### ISSUE TITLE
Add an option to delete hydrogen from the Styles

### ISSUE THREAD

**Issue by julioarvellos** [BradyAJohnston/MolecularNodes#559]
Hello!
I want to suggest a simple feature that allows us to hide/ignore hydrogen atoms in some representations (styles) instead of removing them when importing. 
Basically the idea is to add a switch node inside the node groups so we can decide if we want the hydrogens to be shown.

This is also helpful if you need to show the hydrogens in different parts of the molecule/protein with different representations.

An example of implementation:
![sphere_style](https://github.com/user-attachments/assets/7676b278-2f99-458c-9bca-7967f1d4bee4)

Spheres node group with the checkbox + other nodes that can have this feature
![delete-hydrogen](https://github.com/user-attachments/assets/2f0662b8-7de7-48a4-a274-c6338935ebac)

**Comment by BradyAJohnston** [comment]
I would agree it is a fairly common thing to select for when visualising, but I am unsure about including this in the interface on the node groups. We could maybe include in one of the panels, but it doesn't quite fit in any of them.

**Comment by julioarvellos** [comment]
Hmm maybe an alternative would be a node group that comes native with the structure like the Set Color and Style?
The node group could be ignored if the "Remove Hydrogen" checkbox is active when importing new structures.

![Screenshot_1](https://github.com/user-attachments/assets/29147607-dd6b-4cbe-9a25-4306493b7222)
![Screenshot_2](https://github.com/user-attachments/assets/82b27375-d84e-449a-bd80-f6ec86f3e955)

**Comment by BradyAJohnston** [comment]
I've revisited this and I think having the 'hydrogens' as a toggle for the styles where it is relevant (sphere, ball & stick, surface) is probably a good idea and I'll add it in the upcoming version. Having a node which drops hydrogens is probably a bit overkill when we already have selections  etc
